lib/express/crud/responses/index.js

```diff
@@ -13,23 +50,43 @@ module.exports = {
     res.set('Content-Type', 'application/json').status(204).end()
   },
 
-  badRequestResponse: (req, res, error) => {
-    log.error(error)
-    res.status(400).json({ errors: { message: 'Bad request' }})
+  badRequestResponse: (req, res, errorsList) => {
+    const HttpError = new errors.Http.Http400()
+    errorResponse(req, res, HttpError, errorsList)
   },
 
-  notFoundResponse: (req, res) => {
-    log.error('Document not found')
-    res.status(404).json({ errors: { message: 'Document not Found' }})
+  unauthorizedResponse: (req, res, errorsList) => {
+    const HttpError = new errors.Http.Http401()
+    errorResponse(req, res, HttpError, errorsList)
   },
 
-  unprocessableEntityResponse: (req, res, error) => {
-    log.error(error)
-    res.status(422).json({ errors: { message: 'Unprocessable entity' }})
+  forbiddenResponse: (req, res, errorsList) => {
+    const HttpError = new errors.Http.Http403()
+    errorResponse(req, res, HttpError, errorsList)
   },
 
-  applicationErrorResponse: (req, res, error) => {
-    log.error(error)
-    res.status(500).json({ errors: { message: 'Internal server error' }})
+  notFoundResponse: (req, res, errorsList) => {
+    const HttpError = new errors.Http.Http404()
+    errorResponse(req, res, HttpError, errorsList)
   },
-}
+
+  unprocessableEntityResponse: (req, res, errorsList) => {
+    const HttpError = new errors.Http.Http422()
+    errorResponse(req, res, HttpError, errorsList)
+  },
+
+  lockedResponse: (req, res, errorsList) => {
+    const HttpError = new errors.Http.Http423()
+    errorResponse(req, res, HttpError, errorsList)
+  },
+
+  applicationErrorResponse: (req, res, errorsList) => {
+    const HttpError = new errors.Http.Http500()
+    errorResponse(req, res, HttpError, errorsList)
+  },
+
+  badGatewayResponse: (req, res, errorsList) => {
+    const HttpError = new errors.Http.Http502()
```

Comment:
Use `httpError` — this is instance of the class not a class that goes with capital letter.